api/v1/api_v1_quotas.go

```diff
@@ -29,23 +28,22 @@ func GetIsiQuota(
 	client api.Client,
 	path string) (quota *IsiQuota, err error) {
 
-	// PAPI call: GET https://1.2.3.4:8080/platform/1/quota/quotas
-	// This will list out all quotas on the cluster
+	// PAPI call: GET https://1.2.3.4:8080/platform/1/quota/quotas?path=/path/to/volume
+	// This will list the quota by path on the cluster
```

Comment:
Is this a regression in functionality, because we no longer have a function to get all quotes? Or have we simply addes a new call?

Reply:
This function used to return a quota for a directory and it still returns the same. The way it queried earlier was not right (queries all quotas in a cluster and look for given path - this won't work if the cluster has > 1000 quotas), it's been corrected now.